connect/src/com/telenor/connect/AbstractSdkProfile.java

```diff
@@ -96,4 +115,21 @@ public void failure(RetrofitError error) {
                     }
                 });
     }
+
+    @Override
+    public String getLastAuthState() {
+        return lastAuthState;
+    }
+
+    @Override
+    public void validateTokens(ConnectTokensTO tokens, Date serverTimestamp) {
+        Validator.notNullOrEmpty(tokens.getAccessToken(), "access_token");
```

Comment:
Hehe, we settled it in spoken :) But yeah, sure, i tested the parser, it worked.